.github/pull_request_template.md

```diff
@@ -8,6 +8,7 @@
 - `yarn test:unit:all` should run without errors or warnings
```

Comment:
It seems SSN should be obscured as a password field given its sensitivity, no? Or, even, better all but the last 4 digits are obscured.

Reply:
I'm actually not sure the best practice here, where a search input is being used. We _can_ mask, but it's not part of the current designs. If you think we should, a new ticket is probably the way to go.